src/index.ts

```diff
@@ -27,6 +33,7 @@ import type {
 declare module 'h3' {
   interface H3EventContext {
     i18n?: CoreContext
+    i18nLocale?: LocaleDetector
```

Comment:
`i18nLocale` is for internal use, so we don't want to expose it to H3EventContext. Please don't export it. 🙏

Reply:
The `LocaleDetector` getter needs to bind to event scope. Maybe rename it to `_i18nLocale`, which means that it's private. 🤔

Reply:
> Maybe rename it to _i18nLocale, which means that it's private. 🤔

Look good to me!
You can change to it!

Reply:
Done~